po/ta.po

```diff
@@ -48,7 +48,7 @@ msgstr "Martin Abente Lahaye"
 
 #: data/com.github.tchx84.Flatseal.gschema.xml:15
 msgid "Application ID that was last selected"
-msgstr ""
+msgstr "கடைசியாக தேர்ந்தெடுக்கப்பட்ட விண்ணப்ப ஐடி"
```

Comment:
```suggestion
msgstr "கடைசியாக தேர்ந்தெடுக்கப்பட்ட பயன்பாடு ஐடி"
```

Thanks for the modification, The other strings LGTM. In the above string, விண்ணப்பம் means envelope which also means applications (but business ones) in this case the widely used term for applications is பயன்பாடு. Will open a PR to update the same.